cogs/smallfunctions.py

```diff
@@ -0,0 +1,43 @@
+import discord
+import random
+import #new file storing priv variables
+from discord.ext import commands
+
+# Class responsible for all the small commands run by server members.
+class SmallFunctions(commands.cog):
+
+# Command function for randomized coinflip. Will return a heads or tails.
+    @commands.command()
+    async def random_coinflip(self, ctx):
```

Comment:
instead of random_coinflip do "coinflip" that way it is easier for the user to type in the command

Reply:
at the moment there are two coinflips thats why i have them separated as such with different names

Reply:
can also add aliases for commands if we don't want to change the name but the default coinflip should just be the simple heads/tails vs the game one can be named something different. perhaps we can have a prefix for everything related to the store/currency/game/etc idea